docs/static/images/Cloud-Bigtable.svg

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Is there a stable link to this SVG file on a Google CDN someplace? There seems to be a unversioned [PNG](https://cloud.google.com/images/products/logos/bigtable-symbol.png) but I couldn't find an SVG. That way, docs will always show the same image even if it is updated on your side?